vagrant/Makefile

```diff
@@ -21,6 +21,9 @@ setup.test.only:
 setup.cluster.only:
 	@$(VSSH) setup "make -C /home/vagrant/ansible setup.cluster"
 
-setup.cluster: setup.prep setup.test.only setup.cluster.only
+client1.test:
+	@$(VSSH) client1 "sudo make -C /root/samba-integration-tests tests"
```

Comment:
I am not 100% sure if this is clean: I thought we should trigger the tests from the setup machine. Just like we do the setting up of the cluster and client machine from the setup machine.

Reply:
Also `/root` has to go...

Reply:
The last patch takes the calling off from client.test(where it was finally called through setup) and calls it through the make file instead. Makes it a bit easier to test for developers to trigger the tests by themselves on the client without ssh-ing into the client or the setup.